src/pages/CardNicknameInputPage/index.tsx

```diff
@@ -0,0 +1,75 @@
+import { useEffect, useRef } from 'react';
+
+import Card from '../../components/Card';
+import Spacer from '../../components/common/Spacer';
+import Input from '../../components/common/Input';
+
+import useCardNicknameInputPage from './useCardNicknameInputPage';
+import type { Focus } from '../../components/common/Input';
+import type { CardInfo } from '../../domain/types/card';
+
+import styles from './cardNicknameInputPage.module.css';
+
+const CardNicknameInputPage = () => {
+  const inputRef = useRef<Focus>(null);
+  const {
+    lastRegisteredCard,
+    nickname,
+    handleNicknameSubmit,
+    handleNicknameChange,
+  } = useCardNicknameInputPage();
+  const {
+    cardCompany,
+    cardNumber1,
+    cardNumber2,
+    cardNumber3,
+    cardNumber4,
+    expiredMonth,
+    expiredYear,
+    owner,
+  } = lastRegisteredCard as CardInfo;
+
+  useEffect(() => {
+    inputRef.current?.focus();
+  }, []);
+
+  return (
+    <div className={styles.container} onSubmit={handleNicknameSubmit}>
+      <Spacer height={130} />
+      <h1 className={`${styles.title} text-title`}>
+        카드등록이 완료되었습니다.
+      </h1>
+      <Spacer height={36} />
+      <Card
+        cardCompany={cardCompany}
+        cardNumber1={cardNumber1}
+        cardNumber2={cardNumber2}
+        cardNumber3={cardNumber3}
+        cardNumber4={cardNumber4}
+        expiredMonth={expiredMonth}
+        expiredYear={expiredYear}
+        owner={owner}
+      />
+      <Spacer height={124} />
+      <form className={styles.formContainer}>
+        <label>
+          <Input
+            id="card-nickname"
+            className={styles.nicknameInput}
+            type="text"
+            value={nickname}
+            onChange={handleNicknameChange}
+            placeholder="카드 별칭 입력(선택)"
+            ref={inputRef}
+            aria-label="카드 별칭 입력(선택)"
+          />
```

Comment:
```suggestion
          <Input
            id="card-nickname"
            className={styles.nicknameInput}
            type="text"
            value={nickname}
            onChange={handleNicknameChange}
            placeholder="카드 별칭 입력(선택)"
            ref={inputRef}
          />
```

input에 플레이스 홀더가 있고, label도 존재하고 있기 때문에 aria-label은 여기에서는 꼭 필요하진 않은 것 같아요. 기본 요소들에 대해서 기본적으로 읽어주는 메세지를 선택하는 알고리즘이 존재하고 있어서, 한 번 찾아보셔도 좋을 것 같습니다~! 접근성을 신경써주는 것이 너무 좋와용

Reply:
Implicit native semantics라는 알고리즘이라고 하네요!
`aria-label`을 모든 엘리먼트에 매번 추가해야 할 지 고민이었는데, 서니의 여러 코멘트 덕분에 기준을 점차 잡아나갈수 있을 것 같아요 😎
불필요한 aria-label은 제거했습니다 ✅ [Commit](https://github.com/woowacourse/react-payments/pull/235/commits/61fcbe6a88d3ad031e04e80173f226da82e4dc1e)